static/app/views/replays/list/replayOnboardingPanel.tsx

```diff
@@ -1,5 +1,7 @@
 import styled from '@emotion/styled';
 
+import emptyStateImg from 'sentry-images/spot/replays-empty-state.svg';
```

Comment:
How big is this image? Can we lazy load it?

Reply:
We should lazy load these, but none of the other onboarding screens are doing that. I'll follow up with a PR for all of them.

Reply:
Confirmed that the imports resolve to be public urls, the images are not part of the bundle. So there's nothing to do here.